stackslib/src/chainstate/stacks/boot/signers-voting.clar

```diff
@@ -55,16 +55,14 @@
 (define-read-only (get-tally (reward-cycle uint) (round uint) (aggregate-public-key (buff 33)))
     (map-get? tally {reward-cycle: reward-cycle, round: round, aggregate-public-key: aggregate-public-key}))
 
-(define-read-only (get-signer-slots (signer-index uint) (reward-cycle uint))
-    (let ((height (reward-cycle-to-burn-height reward-cycle)))
-            (ok (at-block
-                (unwrap! (get-block-info? id-header-hash height) err-invalid-burn-block-height)
-                    (get-current-signer-slots signer-index)))))
-
 (define-read-only (get-current-signer-slots (signer-index uint))
```

Comment:
Should this get renamed to get-current-signer-weights? FOr the same reasoning as my prior comment.

Reply:
Same as above -- I'm just using the names that already exist